components/Icons/MdCheckCircleOutlined.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default ({ size = 24, fill }) => (
```

Comment:
Next / the hot reloader does not like those anymore:
> Anonymous arrow functions cause Fast Refresh to not preserve local component state.
> Please add a name to your function, for example:
>
> Before
> export default () => <div />;
> 
> After
> const Named = () => <div />;
> export default Named;

Same with the other two icons.